bin/check_samples.sh

```diff
@@ -26,7 +26,12 @@ ISTIOCTL=bin/istioctl
 go build -o $ISTIOCTL ./istioctl/cmd/istioctl
 
 for f in samples/**/*.yaml; do
-  echo "Validating $f..."
+  if grep -q -e "{{" "$f" ; then
+    echo "Skipping check for helm template $f"
```

Comment:
I wonder if there is some `find` incantation to skip folders with a Chart.yaml :thinking: 

Probably not -- this is fine too.